src/lib/dialog/README.md

```diff
@@ -40,15 +40,12 @@ export class PizzaComponent {
 
   dialogRef: MdDialogRef<PizzaDialog>;
 
-  constructor(
-    public dialog: MdDialog,
-    public viewContainerRef: ViewContainerRef) { }
+  constructor(public dialog: MdDialog) { }
 
   openDialog() {
-    let config = new MdDialogConfig();
-    config.viewContainerRef = this.viewContainerRef;
-
-    this.dialogRef = this.dialog.open(PizzaDialog, config);
+    this.dialogRef = this.dialog.open(PizzaDialog, {
+      role: 'alertdialog'
```

Comment:
This shouldn't be `alertdialog` since that's not appropriate to the content being opened.